config/initializers/oauth2.rb

```diff
@@ -0,0 +1,17 @@
+OAuth2::MACToken.prepend(Module.new do
+  # this is a copy of original method but without using MD5 to
+  # enable FIPS environment compatibility.
+  # see https://github.com/oauth-xx/oauth2/pull/587
+  def header(verb, url)
```

Comment:
I'm not as worried about a test for this method since you noted the exact place you pulled this exact code from and that codebase is fully tested.

Reply:
Yes, also probably we don't use mac token as it is a very old draft standard. At least I hope not used. See linked pull request.